### PR TITLE
All references to an Ubuntu LTS version should include 'LTS'

### PR DESCRIPTION
**Pull request by k-dimple**

### CODE REVIEW

**Review by danpdraper — CHANGES_REQUESTED** [review]
Who is calling for these changes? Marketing?

File: aws/aws-how-to/kubernetes/deploy-ubuntu-pro-cluster.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ You need:
 Prepare the cluster for deployment
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Although Ubuntu Pro for EKS is available as an AMI for Ubuntu 22.04, (see :doc:`deploy-ubuntu-pro-cluster-with-eks-pro-ami`), there is no such EKS related offer available for Ubuntu 20.04.
+Although Ubuntu Pro for EKS is available as an AMI for Ubuntu 22.04 LTS, (see :doc:`deploy-ubuntu-pro-cluster-with-eks-pro-ami`), there is no such EKS related offer available for Ubuntu 20.04 LTS.
```
[the remark this answers is not in the document]
Comment (k-dimple) [reply aws/aws-how-to/kubernetes/deploy-ubuntu-pro-cluster.rst]: Thanks for the catch! I have removed it.

**Comment by k-dimple** [comment]
> Who is calling for these changes? Marketing?

It's in our [branding guidelines](https://docs.ubuntu.com/styleguide/en#branding) which are framed based on inputs from Marketing.

**Review by gjolly — APPROVED** [review]
I like consistency!

**Review by utkarsh2102 — APPROVED** [review]

**Review by danpdraper — APPROVED** [review]

**Pull request merged**